src/platform_impl/linux/x11/window.rs

```diff
@@ -180,13 +181,41 @@ impl UnownedWindow {
         };
 
         // creating
+        let (depth, visual) = match pl_attribs.visual_infos {
+            Some(vi) => (vi.depth, vi.visual),
+            None if window_attrs.transparent == true => {
+                // Find a suitable visual
+                let mut vinfo = MaybeUninit::uninit();
+                if unsafe {
+                    (xconn.xlib.XMatchVisualInfo)(
+                        xconn.display,
+                        screen_id,
+                        32,
+                        TrueColor,
+                        vinfo.as_mut_ptr(),
+                    )
+                } != 0
+                {
+                    let vinfo = unsafe { vinfo.assume_init() };
+                    (vinfo.depth, vinfo.visual)
+                } else {
+                    // There is not a visual that matches the criteria of being `TrueColor` and 32 bits depth,
+                    // and panic or error is not appropiate so just ignore it and continue
+                    (ffi::CopyFromParent, ffi::CopyFromParent as *mut ffi::Visual)
+                }
```

Comment:
I think it's a good idea to log a debug message here like
```rust
debug!("Could not set transparency, because XMatchVisualInfo returned zero for the required parameters");
```

Reply:
Addressed, thank you